include/chttpd.hrl

```diff
@@ -20,3 +20,9 @@
     should_log = true,
     reason
 }).
+
+-define(is_hex(C),
+    (C >= $0 andalso C =< $9) orelse
```

Comment:
do we want a surrounding `()` around the three clauses here so this is safe to include anywhere without unexpected side-effects?